scripts/AbinsModules/Instruments/__init__.py

```diff
@@ -1,5 +1,13 @@
 # flake8: noqa
-from __future__ import (division, print_function)
-from ToscaInstrument import ToscaInstrument
-from TwoDMap import TwoDMap
-from Instrument import Instrument
+from __future__ import (absolute_import, division, print_function)
+# Make sure we can find Instruments...
```

Comment:
@Anders-Markvardsen You're correct, they should not be required. https://docs.python.org/2.5/whatsnew/pep-328.html gives a good overview of this.